android/src/main/java/com/reactnativecommunity/webview/RNCWebViewManager.java

```diff
@@ -215,7 +216,14 @@ public void onDownloadStart(String url, String userAgent, String contentDisposit
 
         RNCWebViewModule module = getModule(reactContext);
 
-        DownloadManager.Request request = new DownloadManager.Request(Uri.parse(url));
+        DownloadManager.Request request;
+        try {
+          request = new DownloadManager.Request(Uri.parse(url));
+        } catch (IllegalArgumentException e) {
+          System.out.println("Unsupported URI, aborting download" + e.toString());
+          e.printStackTrace();
```

Comment:
This is my remaining concern at this point. I think this should be using the Android Logger to log a warning (and pass the exception to it, which the logger accepts as a parameter) rather than using `println` & `printStackTrace`.

Reply:
Could you please elaborate on what you mean by the Android logger? I was able to see this stack trace printed out in Android Studio.

If you mean RNLog from the previous discussion, we would need a React Native version bump first.

Reply:
No, I don't mean RNLog (I also don't mean `FLog` either... I'm not familiar with that one and unsure why it's used). I mean just `Log.w`. If you search the code you'll find examples, such as https://github.com/react-native-webview/react-native-webview/blob/b3cba2bb8e7d2ddca4cf852ceef6169c82699a33/android/src/main/java/com/reactnativecommunity/webview/RNCWebViewManager.java#L1003

So I mean something like (I didn't notice GitHub supported "suggestions" 'til now):
```suggestion
          Log.w(TAG, "Unsupported URI, aborting download", e);
```

I admit I haven't actually tried using `println` on Android (I've always used `Log`), so I don't know exactly how they compare, but my understanding from what I read is that `println`/`printStackTrace` outputs in a way that could be easy to miss (e.g., if you filter to only your app, those lines might not show up?).

Reply:
TIL. Thanks. I took the liberty of cleaning up a few of the other logs.

Reply:
Thanks! _Normally_ I'd vote to have other cleanup in a separate PR, but this looks pretty small & straightforward... especially switching to `TAG` uses